harness/reporters/reporter.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reporters
+
+import (
+	"time"
+)
+
+const (
+	TestResultFail TestResult = "FAIL"
+	TestResultSkip TestResult = "SKIP"
+	TestResultPass TestResult = "PASS"
+)
+
+type TestResult string
+
+type Reporters []Reporter
+
+func (reps *Reporters) ReportTest(name string, result TestResult, duration time.Duration, b []byte) {
```

Comment:
Is there a reason this is a pointer receiver? Since it's interacted with via `append` below, I don't think it needs to be a `*`